tests/behaviour/steps/inspectcli.py

```diff
@@ -59,9 +90,11 @@ def step_impl(context):
 def step_impl(context):
     metadata_printer = MetadataPrinter(
         context.csvw_type,
+        context.csvw_shape,
         context.csvw_metadata_rdf_graph,
         context.csvw_metadata_json_path,
     )
+    # TODO: Remove below once all the tests are updated to not match strings
```

Comment:
Do we need this still?

Reply:
Yes, this is related to the old text matching style behave tests.